sdk/storage/Azure.Storage.Blobs/tests/ClientSideEncryptionTests.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.Cryptography;
+using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Keys.Cryptography;
+using Azure.Storage.Cryptography;
+using Azure.Storage.Cryptography.Models;
+using Azure.Storage.Test.Shared;
+using Azure.Storage.Tests.Shared;
+using NUnit.Framework;
+
+namespace Azure.Storage.Blobs.Test
+{
+    public class ClientSideEncryptionTests : BlobTestBase
+    {
+        private const string ThrowawayAlgorithmName = "blah";
+        public ClientSideEncryptionTests(bool async, BlobClientOptions.ServiceVersion serviceVersion)
+            : base(async, serviceVersion, null /* RecordedTestMode.Record /* to re-record */)
+        {
+        }
+
+
+        #region Utility
```

Comment:
nit: I'm not sure what this region means. besides it's not closed.

Reply:
It is closed, write above the first test.

Region is intended to separate tests from methods that help run the tests.

Reply:
okay. I think vertical placement (tests near tests, aids near aids) is good enough. I'd avoid `Utility` naming if you want to keep region